src/shared/components/deletion-confirmation/DeletionConfirmation.scss

```diff
@@ -0,0 +1,18 @@
+@import 'src/styles/common';
+
+.deleteConfirmationContainer {
+  display: flex;
+  align-items: center;
+  justify-content: var(--container-justify-content, flex-end);
```

Comment:
You shouldn't need this CSS rule.

Reply:
But what if we want the text to be at the start, do we not need to overwrite this?

Reply:
Agreed this is not needed as a variable.